helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -19,19 +19,21 @@
  * under the License.
  */
 
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Maps;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
 import org.apache.helix.HelixException;
 import org.apache.helix.HelixProperty;
 import org.apache.helix.ZNRecord;
 import org.apache.helix.api.config.HelixConfigProperty;
 import org.apache.helix.api.config.StateTransitionThrottleConfig;
 import org.apache.helix.api.config.StateTransitionTimeoutConfig;
 
+import java.util.ArrayList;
```

Comment:
Would it be great to put names in ASCII sort order?

Reply:
The current order is the ASCII sort order. Or could you please specify what kind of order you are suggesting?